windows/src/README.md

```diff
@@ -54,13 +54,14 @@ environment.
    Environment variable `HELP_KEYMAN_COM` needs to be set to the root of the local 
    help.keyman.com git repository.
 
-Note: by default, the version generated will be 10.0.700.0. You will not be able to
-install it over a later version of Keyman, and will need to uninstall and reinstall.
+Note: by default, the version number generated may vary from the current release version. 
+You will not be able to install it over a later version of Keyman, and will need to 
+uninstall and reinstall.
 
 ## Installing Keyman
 
 These steps are only required the first time you install Keyman:
-1. Install release versions of Keyman 10.0 and Keyman Developer 10.0.
+1. Install release versions of Keyman Desktop and Keyman Developer
   * Download the [latest official builds (alpha)](https://keyman.com/beta/) and run the installers.
```

Comment:
Do we want these alpha/beta references & links to stick with stable?

Reply:
Probably should note that you should download the latest release from the branch from which you are working (e.g. for master, use latest alpha).

Reply:
Will fix in next push